src/main/resources/reference.conf

```diff
@@ -1312,6 +1340,32 @@ resourceTypes = {
     reuseIds = false
     authDomainConstrainable = true
   }
+  snapshot-builder-request = {
+    actionPatterns = {
+      get = {
+        description = "Get or list the snapshot builder request"
+      }
+      update = {
+        description = "Update the snapshot builder request"
+      }
+      delete = {
+        description = "Delete the snapshot builder request"
+      }
+      approve = {
+        description = "Approve or reject the snapshot builder request"
+      }
+    }
+    ownerRoleName = "owner"
+    roles = {
+      owner = {
+        roleActions = ["get", "update", "delete"]
```

Comment:
I didn't notice it before, but it looks like TDR has a convention of including the resource type in the action name, e.g. `update_snapshot`, `read_dataset`, etc. And there is a new action above, `create_snapshot_request`. Although I also see `delete` and `create_snapshot_request` is a `snapshot` action so it makes sense that it has the type in its name.

Maybe there's nothing to do here but I wonder if we should be trying to make the action names more consistent, and if so, what convention should we use.

Reply:
I think we should use the shorter name. It is equivalent to `snapshot.delete` rather than `snapshot.delete_snapshot`. One of those has significant redundant information.